tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java

```diff
@@ -110,6 +111,33 @@ public void initialize(ConfigurableApplicationContext configurableApplicationCon
 
     }
 
+    private JdbcDatabaseContainer createDatabaseContainer()
+    {
+        switch (getImagesConfig().getDatabaseType())
+        {
+            case POSTGRESQL_DB:
+                setupDatabaseOptions("org.postgresql.Driver", "jdbc:postgresql://postgres:5432/alfresco");
```

Comment:
The first string could be added to enum values, don't know about the second. To be considered.

Reply:
If both are set in an enum, you could take it out of the switch, call setupDatabaseOptions(DatabaseType) and simply use retrieved values